ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -80,6 +80,14 @@ Item {
             isOnline: model.onlineStatus
             icon.color: Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(model.id)]
             ringSettings.ringSpecModel: Utils.getColorHashAsJson(model.id)
+            components: [
+                StatusIcon {
+                    anchors.verticalCenter: parent.verticalCenter
+                    visible: model.isAdmin
+                    source: Style.svg("crown")
+                    color: Theme.palette.inputColor
```

Comment:
Not sure what `inputColor` is, but if we want black in light mode and white in darkmode, we should probably be using `directColor1` to stay consistent..

Reply:
that fair :) I will fix

Reply:
fixed